utilities/rpp-unittests/HOST_NEW/testAllScript_now.sh

```diff
@@ -0,0 +1,433 @@
+
+# for logging
+if [ $test_type -eq 1 ]; then
+    rm -rvf "$cwd/../OUTPUT_PERFORMANCE_LOGS_HOST_NEW"/*
+    mkdir "$cwd/../OUTPUT_PERFORMANCE_LOGS_HOST_NEW"
+    LOGGING_FOLDER="$cwd/../OUTPUT_PERFORMANCE_LOGS_HOST_NEW"
+fi
+
+
+# Images for unique functionalities
+DEFAULT_FAST_CORNER_DETECTOR_IMAGES="$cwd/../TEST_IMAGES/fast_corner_detector"
+DEFAULT_HARRIS_CORNER_DETECTOR_IMAGES="$cwd/../TEST_IMAGES/harris_corner_detector"
+DEFAULT_HOUGH_LINES_IMAGES="$cwd/../TEST_IMAGES/hough_lines"
+DEFAULT_HOG_IMAGES="$cwd/../TEST_IMAGES/hog"
+num_iterations=100
+pkd_layout=0
+pln3_layout=1
+pln1_layout=2
+
+# <<<<<<<<<<<<<< PRINTING THE TEST TYPE THAT USER SPECIFIED >>>>>>>>>>>>>>>>>>>>>>>>>>>>
+if [ $test_type -eq 0 ]; then
+    printf "\n UNIT TESTING..\n"
+fi
+if [ $test_type -eq 1 ]; then
+    printf "\n PERFORMANCE TESTING..\n"
+fi
+
+# <<<<<<<<<<<<<< DEFAULT SOURCE AND DESTINATION FOLDERS (NEED NOT CHANGE) >>>>>>>>>>>>>>
+
+
+
+
+
+# <<<<<<<<<<<<<< FOR MANUAL OVERRIDE, JUST REPLACE AND POINT TO THE SOURCE AND DESTINATION FOLDERS HERE >>>>>>>>>>>>>>
+SRC_FOLDER_1="$DEFAULT_SRC_FOLDER_1"
+SRC_FOLDER_2="$DEFAULT_SRC_FOLDER_2"
+DST_FOLDER="$DEFAULT_DST_FOLDER"
+# <<<<<<<<<<<<<< FOR MANUAL OVERRIDE, JUST REPLACE AND POINT TO THE SOURCE AND DESTINATION FOLDERS HERE >>>>>>>>>>>>>>
+
+
+
+
+
+# <<<<<<<<<<<<<< EXECUTION OF ALL FUNCTIONALITIES (NEED NOT CHANGE) >>>>>>>>>>>>>>
+
+directory_name_generator() {
+
+    AFFINITY=$1
+    TYPE=$2
+    CASE=$3
+
+    if [[ "$case" -lt 15 ]]
+    then
+        FUNCTIONALITY_GROUP="image_augmentations"
+    elif [[ "$case" -lt 20 ]]
+    then
+        FUNCTIONALITY_GROUP="statistical_functions"
+    elif [[ "$case" -lt 29 ]]
+    then
+        FUNCTIONALITY_GROUP="geometry_transforms"
+    elif [[ "$case" -lt 36 ]]
+    then
+        FUNCTIONALITY_GROUP="advanced_augmentations"
+    elif [[ "$case" -lt 40 ]]
+    then
+        FUNCTIONALITY_GROUP="fused_functions"
+    elif [[ "$case" -lt 42 ]]
+    then
+        FUNCTIONALITY_GROUP="morphological_transforms"
+    elif [[ "$case" -lt 49 ]]
+    then
+        FUNCTIONALITY_GROUP="color_model_conversions"
+    elif [[ "$case" -lt 56 ]]
+    then
+        FUNCTIONALITY_GROUP="filter_operations"
+    elif [[ "$case" -lt 65 ]]
+    then
+        FUNCTIONALITY_GROUP="arithmetic_operations"
+    elif [[ "$case" -lt 69 ]]
+    then
+        FUNCTIONALITY_GROUP="logical_operations"
+    elif [[ "$case" -lt 79 ]]
+    then
+        FUNCTIONALITY_GROUP="computer_vision"
+    else
+        FUNCTIONALITY_GROUP="miscellaneous"
+    fi
+
+    DST_FOLDER_TEMP="$DST_FOLDER""/rpp_""$AFFINITY""_""$TYPE""_""$FUNCTIONALITY_GROUP"
+}
+
+if [[ "$1" -lt 0 ]] | [[ "$1" -gt 86 ]]; then
+    echo "The starting case# must be in the 0:86 range!"
+    echo
+    echo "The rawLogsGenScript.sh bash script runs the RPP performance testsuite for AMDRPP functionalities in HOST/OCL/HIP backends."
+    echo
+    echo "Syntax: ./rawLogsGenScript.sh <S> <E> <U>"
```

Comment:
rename ./rawLogsGenScript.sh to testAllScript.sh . Applicable to all such instances
Add info for TEST_TYPE parameter in the existing description as 
./testAllScript.sh <S> <E> <U> <T> 

Applicable to all such instances where description for parameter is needed